packages/components/src/components/overflow-menu/overflow-menu.hbs

```diff
@@ -11,21 +11,25 @@
     <span class="{{@root.prefix}}--assistive-text">Overflow</span>
     {{ carbon-icon 'OverflowMenuVertical16' class=(add @root.prefix '--overflow-menu__icon') }}
   </button>
-  <ul class="{{@root.prefix}}--overflow-menu-options" tabindex="-1" role="menu"
-    aria-labelledby="{{idSuffix.default}}-trigger" data-floating-menu-direction="{{direction}}"
-    id="{{idSuffix.default}}">
-    {{#each items}}
-    <li
-      class="{{@root.prefix}}--overflow-menu-options__option {{#if disabled}} {{@root.prefix}}--overflow-menu-options__option--disabled {{/if}} {{#if danger}} {{@root.prefix}}--overflow-menu-options__option--danger {{/if}}">
-      <button class="{{@root.prefix}}--overflow-menu-options__btn" role="menuitem" {{#if title}} title="{{title}}"
-        {{/if}} {{#if primaryFocus}} data-floating-menu-primary-focus {{/if}} {{#if disabled}} disabled {{/if}}>
-        <span class="{{@root.prefix}}--overflow-menu-options__option-content">
-          {{label}}
-        </span>
-      </button>
-    </li>
-    {{/each}}
-  </ul>
+  <div class="{{@root.prefix}}--overflow-menu-wrapper">
```

Comment:
Similar question to modal - Would you put the focus trap element next to `</ul>` instead?